src/dlgTriggerEditor.cpp

```diff
@@ -815,6 +816,7 @@ void dlgTriggerEditor::slot_item_selected_search_list(QTreeWidgetItem* pItem)
                     mpSourceEditorEdbee->setFocus();
                     controller->setAutoScrollToCaret(edbee::TextEditorController::AutoScrollWhenFocus);
                     controller->moveCaretTo(pItem->data(0, PatternOrLineRole).toInt(), pItem->data(0, PositionRole).toInt(), false);
+                    controller->setAutoScrollToCaret(edbee::TextEditorController::AutoScrollNever);
```

Comment:
We should probably extract a function out of the above at some point...

Reply:
Agreed